standalone/Dockerfile

```diff
@@ -78,7 +78,7 @@ RUN adduser --uid 704704 --system --group --gecos 'tor user' tor \
     && useradd --uid 7012348 --system -g joinmarket --comment 'ob-watcher user' jm-ob \
     && sed "s|http://|https://|" /etc/apt/sources.list \
     && apt-get update \
-    && apt-get install -qq --no-install-recommends --no-install-suggests -y gnupg curl apt-transport-https ca-certificates \
```

Comment:
As of apt 1.5, the package apt-transport-https is no longer required because https:// is supported out of the box.
Reference: https://packages.debian.org/bullseye/apt-transport-https
> This is a dummy transitional package - https support has been moved into the apt package in 1.5. It can be safely removed."
Apt is currently at 2.6.